charts/tidb-operator/templates/admission/admission-webhook-registration.yaml

```diff
@@ -38,12 +38,12 @@ metadata:
     helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+"  "_" }}
 webhooks:
   - name: stsadmission.tidb.pingcap.com
-    {{- if semverCompare ">=1.15-0" .Capabilities.KubeVersion.GitVersion }}
     objectSelector:
       matchLabels:
         "app.kubernetes.io/managed-by": "tidb-operator"
-    {{- end }}
-    failurePolicy: {{ .Values.admissionWebhook.failurePolicy.validation | default "Ignore" }}
+    admissionReviewVersions: ["v1beta1"]
```

Comment:
Refer to the doc here https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/, why we do not have `v1` in the versions?

Reply:
because we use the framework "github.com/openshift/generic-admission-server" which only process v1beta1 message